spec/support/content_store_helpers.rb

```diff
@@ -48,4 +48,15 @@ def content_store_throws_exception_for(path, exception)
   def honours_content_store_ttl
     expect(response.headers["Cache-Control"]).to eq("max-age=#{15.minutes.to_i}, public")
   end
+
+  def stub_homepage_content_item(links: [])
```

Comment:
Minor not blocking:

I can't imagine why we'd ever have a test in this repo where we needed to check what happens if the content of popular links is a certain thing - there's no presentation logic that depends on the type of link etc. So maybe we should just use the existing `content_store_has_example_item`

Reply:
The only test that really depends on the links existing is a request test, which thinking about it should probably be a system test.